CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * `Hash#slice` ignores non existing keys.
 
+# 0.8.0
+
+See https://github.com/svenfuchs/i18n/releases/tag/0.8.0.beta1
```

Comment:
I've just added https://github.com/svenfuchs/i18n/releases/tag/v0.8.0 to the releases list. Could you please update the link here to that one?